project_task_material_stock/data/data.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--
+        Copyright 2015 Tecnativa - Sergio Teruel
+        Copyright 2015 Tecnativa - Carlos Dauden
+        Copyright 2016-2017 Tecnativa - Vicent Cubells
+        License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+    -->
+    <data>
+        <!-- Sequence new picking type -->
+        <record model="ir.sequence" id="project_task_material_sequence">
+            <field name="name">Project Task Material Sequence</field>
+            <field name="active">1</field>
+            <field name="prefix">CONSUM/</field>
+            <field name="padding">5</field>
+            <field name="number_increment">1</field>
+            <field name="implementation">standard</field>
+        </record>
+        <!-- New Picking type -->
+        <record model="stock.picking.type" id="project_task_material_picking_type">
+            <field name="name">Project Task Material Operations</field>
+            <field name="code">outgoing</field>
+            <field name="sequence_code">PTM</field>
+            <field name="sequence_id" ref="project_task_material_sequence" />
+            <field name="warehouse_id" ref="stock.warehouse0" />
+            <field name="default_location_src_id" ref="stock.stock_location_stock" />
+            <field
+                name="default_location_dest_id"
+                ref="stock.stock_location_customers"
+            />
+        </record>
```

Comment:
On v13, this is not hardcoded on data, but populated on this method:

https://github.com/odoo/odoo/blob/b78ab99d1426d2fbabaa5b272ee9e42fc1a66113/addons/stock/models/stock_warehouse.py#L262

and here you inject a new picking type:

https://github.com/odoo/odoo/blob/b78ab99d1426d2fbabaa5b272ee9e42fc1a66113/addons/stock/models/stock_warehouse.py#L830

But I'm doubting if a new picking should be populated instead of using only `stock.move` records without picking. It's the technique used by `mrp` module. Even more, they present in the production form directly `stock.move` and  `stock.move.line` records to manipulate the quantities and products to be consumed or to produce.

I think we should get rid off the model `project.task.material`, and use instead `stock.move` / `stock.move.line` as the way to add products and quantities.

Check how it's done on `mrp`:

https://github.com/odoo/odoo/blob/b78ab99d1426d2fbabaa5b272ee9e42fc1a66113/addons/mrp/models/mrp_production.py#L168-L178

Reply:
Finally, the suggested changes have not been introduced in this addon, therefore this MR would be ready for review.

A new addon https://github.com/OCA/project/pull/905 has been created with these comments using only stock moves.